azure/biganimal-csp-setup

```diff
@@ -328,15 +329,29 @@ check_custom_role()
   fi
 }
 
+wait_for_spn()
+{
+  spn_exist=$(az ad sp list --spn "${app_id}" | jq -r 'length')
+  if (( spn_exist > 0 )); then
+    echo "Updating Azure AD Service Principal application ${app_id} and configuring its access to Azure resources in subscription ${subscription}..."
+    spn_role=$(az role assignment create --assignee "${app_id}" --role "${role_name}" --scope /subscriptions/"${subscription}" --only-show-errors)
```

Comment:
I am concerned that if the role does not exist, the script will exit here with no hint. I will test it.